pontoon/sync/vcs/models.py

```diff
@@ -563,11 +565,12 @@ def get_or_set_project_files(self, locale_code):
             ProjectFiles(locale_code, [self.parsed_configuration]),
         )
 
-    def l10n_path(self, locale, reference_path):
+    def l10n_path(self, locale, absolute_resource_path):
         """
         Return l10n path for the given locale and reference path.
         """
         project_files = self.get_or_set_project_files(locale.code)
+        reference_path = locale_to_source_path(absolute_resource_path)
```

Comment:
`l10n_path` is only ever called with p-c on, so this should be here anymore, I think.
Or it should at least be guarded by an `if`.